server/repository/src/db_diesel/cold_storage_type.rs

```diff
@@ -96,11 +96,9 @@ impl<'a> ColdStorageTypeRepository<'a> {
         let mut query = cold_storage_type_dsl::cold_storage_type.into_boxed();
         // Any cold storage types that don't have temperature set (OdegC to 0degC default value) are invalid => filter out
 
-        query = query.filter(
-            cold_storage_type_dsl::min_temperature
-                .ne(0.0)
-                .and(max_temperature.ne(0.0)),
-        );
+        query = query.filter(not(cold_storage_type_dsl::min_temperature
+            .eq(0.0)
+            .and(max_temperature.eq(0.0))));
```

Comment:
can't this just be 

```suggestion
        query = query.filter(cold_storage_type_dsl::min_temperature
            .eq(0.0)
            .and(max_temperature.ne(0.0)));
```


?

Reply:
I don't think so? Did you try it? I'm happy with the solution I have...

Reply:
Yeah I tried both works XD. So I'm happy with either 🤷🏻